src/modules/copyFile.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const fs = require('fs');
+
+const copyFile = (sourcePath, destPath) => {
+  if (sourcePath === destPath) {
+    return;
+  }
+
+  try {
+    const data = fs.readFileSync(sourcePath, 'utf-8');
+
+    fs.writeFileSync(destPath, data, 'utf-8');
```

Comment:
Not all files are `utf-8`, don't pass this param and work with raw buffers
```suggestion
    const data = fs.readFileSync(sourcePath);

    fs.writeFileSync(destPath, data);
```

Reply:
Try copying an executable file, it will not work afterwards